src/main/java/com/autilus/hw0406/task2/Car.java

```diff
@@ -0,0 +1,73 @@
+package com.autilus.hw0406.task2;
+
+import java.io.*;
+
+public class Car implements Serializable {
+    private String name;
+    private String model;
+    private int year;
+    private int maxSpeed;
```

Comment:
you use only primitive type here
how about other Objects? (person and car / car and wheel)?

Reply:
added)